roles/keystone/tasks/main.yml

```diff
@@ -80,6 +87,17 @@
   service: name=keystone state=stopped
   when: database_create.changed or force_sync|default('false')|bool
   tags: db-migrate
+  
+- name: disable keystone apache site before db sync
+  apache2_site: name=keystone state=disabled
+  when: database_create.changed or force_sync|default('false')|bool
+  tags: db-migrate
```

Comment:
Does the apache front end need to be killed off? Is a 404 better than a 502? (is it a 404 if the front end is shut off?)